services/ui-src/src/forms/mcpar/mcpar.json

```diff
@@ -1464,6 +1464,7 @@
             },
             "dashboardTitle": "Report program characteristics & enrollment for each plan",
             "drawerTitle": "Report program characteristics & enrollment for",
+            "drawerReminder": "Complete all fields and select the Save & close button to save this section.",
             "missingEntityMessage": [
```

Comment:
added this comment on Aileen's original PR but I'll add it here: should it be `Save & Close` instead of `Save & close`?

Reply:
Probably, although lowercase is what's in the mockup in the ticket. @ailZhou please follow up with design and see what they say.

Reply:
@ailZhou This ticket is also probably ready for design review. Follow the [instructions i left on your other PR here](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/pull/11166#issuecomment-1478629850) to update the ticket and notify them.

Reply:
followed up with design, it's "Save & close" and they also reviewed the PR saying it looks good